.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           java-version: '12.x'
       - uses: subosito/flutter-action@v1
         with:
-          channel: 'stable'
+          channel: 'dev'
```

Comment:
Does that means that people will require the dev channel of Flutter to use Flam?

Reply:
No, they will need at least `1.18.0-6.0.pre`. I don't know what branch that is in at the moment. But this is a requirement introduced by flutter_svg 0.18.0, so we can't upgrade if we don't want that.

Reply:
We usually, keep Flame following the beta channel, let me confirm which version is the latest beta.

Reply:
I just changed to dev as that's the one I use. I will try beta to see if the build works.